src/UsEastCertificateStack.ts

```diff
@@ -27,17 +27,11 @@ export class UsEastCertificateStack extends Stack {
 
   createCertificate() {
     const subdomain = Statics.subDomain(this.branch);
-    const cspSubdomain = Statics.subDomain(this.branch);
+    const cspSubdomain = Statics.cspSubDomain(this.branch);
```

Comment:
Hoort deze wijziging ook bij deze change?

Reply:
Dit was een fix in productie omdat de deployment fout ging, ik heb daarna de branch naar accp gemerged. Gek genoeg ziet die dit als een aanpassingen, in productie wordt cspSubDomain al gebruikt namelijk.

Reply:
Ah, ja dat herinner ik me nog. Waarom dingen die al gelijk zijn soms als wijziging meegaan snap ik ook nooit helemaal.